Automattic/jetpack
# Action bar: try iframe approach

#### Changes proposed in this Pull Request:

Variation on #25447

Injects the action bar via an iframe from widgets.wp.com, with the goal of having necessary authentication for following and likes.

#### Other information:


#### Jetpack product discussion

pe7F0s-1U-p2

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

Apply D86901-code and sandbox the specified domains

* Switch to this branch and build the project: `jetpack install --all`
* Activate the Action bar module from /wp-admin/admin.php?page=jetpack_modules
* Open a single post page on the front-end, and you should see the action bar
* Click on the star to like the post

More functionality in progress...
